clients/drcachesim/common/memtrace_stream.h

```diff
@@ -93,6 +93,8 @@ class memtrace_stream_t {
         SCHED_STAT_DIRECT_SWITCH_ATTEMPTS,
         /** Count of #TRACE_MARKER_TYPE_DIRECT_THREAD_SWITCH attempts that succeeded. */
         SCHED_STAT_DIRECT_SWITCH_SUCCESSES,
+        /** Count of switches where the new input's prior core != its new core. */
```

Comment:
This comment describes how the counter works, but I think more clarity on what it means would be helpful.  This is counting migrations from the view of the output, while I tend to think of them more from the view of the input (e.g. how many times each input changed cores).  Can you expand the comment to clarify this distinction more?  Something like "Counts the number of times an input switches from another core to this core, e.g. the number of input migrations to this core."

Reply:
Switched to your description.